From 83f08ed9f3cd658e07fe25c9e22887c52f2379a5 Mon Sep 17 00:00:00 2001
From: Jason Abele <jason@jasonabeleconsulting.com>
Date: Wed, 22 Jul 2015 17:33:26 -0700
Subject: [PATCH 1/2] rtl8723bs: remove unused wext wireless_handlers

The wireless_handlers member has been removed from net_device in recent
kernels and is not required by this driver.  Remove it to fix
compilation.
---
 os_dep/os_intfs.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/os_dep/os_intfs.c b/os_dep/os_intfs.c
index b30c2a0..13d0000 100644
--- a/os_dep/os_intfs.c
+++ b/os_dep/os_intfs.c
@@ -564,7 +564,6 @@ struct net_device *rtw_init_netdev(struct adapter *old_padapter)
 
 	/* pnetdev->tx_timeout = NULL; */
 	pnetdev->watchdog_timeo = HZ*3; /* 3 second timeout */
-	pnetdev->wireless_handlers = (struct iw_handler_def *)&rtw_handlers_def;
 
 	/* step 2. */
 	loadparam(padapter, pnetdev);
-- 
1.9.1

